content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,165 @@
+
+{/*TODO: ![🖼 Include a DO example with image to reinforce message]()*/}
+
+{/*TODO: reference the case where there's a declaratively saved form on a page with inherently saved inputs (such as user Profile settings)*/}
+
+#### Common places for a submit button
+
+Default to placing the submit button at the bottom left of the form. Field labels and inputs are left-aligned and run from top-to-bottom, so users would naturally move their eyes down and to the left.
+
+{/*TODO: ![🖼 Include a DO example with image to reinforce message]()*/}
+
+If the save button is used in a dialog, place the button at the bottom right of the dialog. Right-aligned dialog buttons feel familiar because it's a very common pattern. For example: Windows, macOS, and Android.
+
+{/*TODO: ![🖼 Include a DO example with image to reinforce message]()*/}
+
+If the save button in the dialog has a label that makes the button almost as wide as the dialog, you may opt to make the dialog action buttons full-width. If there is a secondary action button, it must also be full-width.
+
+{/*TODO: ![🖼 Include a DO example with image to reinforce message]()*/}
+
+If the button is used to send a message such as a comment on an issue, right-align the button below the text area. This is where GitHub has traditionally placed the "Submit" button for issues and pull requests, and it's common to place a "send" button to the right. For example: iMessage, Facebook (mobile web), Twitter, Reddit.
+
+{/*TODO: Make sure right-aligned send buttons are ok for right-to-left languages*/}
+{/*TODO: ![🖼 Include a DO example with image to reinforce message]()*/}
+
+{/*
+TODO:
+1. Be more specific about when a button should be placed somewhere besides the bottom-right. Define alternative patterns and when to use them.
+
+2. Consider how we might define more patterns to keep the "Save" button easily accessible on long forms:
+- add save and cancel buttons on top AND bottom?
+- use a sticky header with save and cancel buttons?
+*/}
+
+To minimize UI, some content may be edited inline without taking the user to a separate flow. An input used for inline editing should have a save button placed very close to visually group the input with the button.
+
+![Save button is placed to the right of the input used to edit the PR title](https://user-images.githubusercontent.com/2313998/150868914-6af36067-39a6-4fec-adb9-518c5abd595d.gif)
+
+If you encounter a case where your submit button would not be intuitive or easily accessible using any of the placement options described above, then consider alternatives and optinally propose a new Primer pattern.
+
+### State
```

Comment:
Because state is so small, I'm wondering if you should move it to before placement.